crates/storage_models/src/enums.rs

```diff
@@ -464,6 +464,91 @@ pub enum PaymentMethod {
     BankDebit,
 }
 
+// #[cfg(feature = "payouts")]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::EnumString,
+    frunk::LabelledGeneric,
+)]
+#[router_derive::diesel_enum(storage_type = "pg_enum")]
+#[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
+pub enum PayoutStatus {
+    Success,
+    Failed,
+    Cancelled,
+    Pending,
+    Ineligible,
+    RequiresCreation,
+    RequiresPayoutMethodData,
+    #[default]
+    RequiresFulfillment,
```

Comment:
should default be `pending`?

Reply:
`RequiresCreation` will be the default, which is added in other PR